### PR TITLE
routeros_facts: prevent crash of module when ipv6 package is not installed

### PR DESCRIPTION
**Pull request by heuels**
##### SUMMARY

Backport of ansible-collections/community.general#39

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`routeros_facts`

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/64958-routeros-facts-ipv6.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - routeros_facts - Prevent crash of module when ``ipv6`` package is not installed
+  - routeros_facts - Unit tests for the case when ``ipv6`` package is not installed
```
Comment (felixfontein) [inline changelogs/fragments/64958-routeros-facts-ipv6.yml]: You don't need a changelog fragment for the unit tests.

Comment (heuels) [reply changelogs/fragments/64958-routeros-facts-ipv6.yml]: Oh, OK. Removed it, thanks!

**Review by felixfontein — APPROVED** [review]
Looks good (and code change is idential to change in community.general). I've tested the community.general change on my hardware, it doesn't break anything for users where this module worked before.

**Comment by heuels** [comment]
@mattclay, thanks!

**Pull request merged**